src/libraries/Directory.Build.props

```diff
@@ -15,7 +17,7 @@
     <StrongNameKeyId>MicrosoftAspNetCore</StrongNameKeyId>
     <IsAspNetCoreApp>true</IsAspNetCoreApp>
     <!-- Temporarily ignore nit warning/errors (e.g. extra whitespace) from Extensions projects -->
-    <NoWarn Condition="!$(MSBuildProjectName.EndsWith('.Tests'))">$(NoWarn);SA1129;SA1028;SA1027;SA1121;CA1200</NoWarn>
+    <NoWarn Condition="!$(MSBuildProjectName.EndsWith('.Tests'))">$(NoWarn);SA1129;SA1028;SA1027;SA1121;CA1200;SA1000;CA1507;CA1802;CA1825</NoWarn>
```

Comment:
For the curious (like me)
SA1129: DoNotUseDefaultValueTypeConstructor
SA1028: CodeMustNotContainTrailingWhitespace
SA1027: UseTabsCorrectly
SA1121: UseBuiltInTypeAlias
CA1200: Avoid using cref tags with a prefix
SA1000 : KeywordsMustBeSpacedCorrectly
CA1507: Use nameof in place of string
CA1802: Use Literals Where Appropriate
CA1825: Avoid zero-length array allocations

Most of these should be easy to come back and fix at once.